### ISSUE TITLE
Synthetics homepage summary table can't be copy/pasted to dashboard

### ISSUE THREAD

**Issue by peterschretlen** [grafana/synthetic-monitoring-app#954]
**What happened**:

Sometimes there's a need to tweak the summary table panel on the synthetics home page, for example to:
- Change the color scheme or thresholds
- Add a column
- Change how latency, uptime, or reachability are displayed or calculated

The typical workflow to do this is to "Copy JSON" and then on a new panel, Inspect > Panel JSON and paste the JSON in. 
This doesn't work for this panel.  It's possible to copy the JSON, and paste it. Doing that gives the following error in the browser console:

```
utils.js:32 Uncaught Error: Cannot clone a SceneObject with a SceneObjectRef in state
    at it (utils.js:32:13)
    at He (utils.js:21:23)
    at Qa.clone (SceneObjectBase.js:226:12)
    at Object.it [as cloneSceneObjectState] (utils.js:29:36)
    at go.onEnterEditMode (DashboardScene.tsx:249:37)
    at onApplyChange (InspectJsonTab.tsx:123:17)
    at Object.Ut (react-dom.production.min.js:54:317)
    at mt (react-dom.production.min.js:54:471)
    at At (react-dom.production.min.js:55:35)
    at Wc (react-dom.production.min.js:105:68)
    at jc (react-dom.production.min.js:106:380)
    at react-dom.production.min.js:117:104
    at Ac (react-dom.production.min.js:274:42)
    at Xt (react-dom.production.min.js:52:375)
    at Mu (react-dom.production.min.js:109:469)
    at Ci (react-dom.production.min.js:74:59)
    at zr (react-dom.production.min.js:73:73)
    at Tt.invokeTask (zone.js:402:33)
    at xe.runTask (zone.js:159:47)
    at Bt.invokeTask [as invoke] (zone.js:483:34)
    at D (zone.js:1138:18)
    at X (zone.js:1181:33)
    at HTMLDivElement.ee (zone.js:1202:16)
it @ utils.js:32
He @ utils.js:21
clone @ SceneObjectBase.js:226
it @ utils.js:29
onEnterEditMode @ DashboardScene.tsx:249
onApplyChange @ InspectJsonTab.tsx:123
Ut @ react-dom.production.min.js:54
mt @ react-dom.production.min.js:54
At @ react-dom.production.min.js:55
Wc @ react-dom.production.min.js:105
jc @ react-dom.production.min.js:106
(anonymous) @ react-dom.production.min.js:117
Ac @ react-dom.production.min.js:274
Xt @ react-dom.production.min.js:52
Mu @ react-dom.production.min.js:109
Ci @ react-dom.production.min.js:74
zr @ react-dom.production.min.js:73
invokeTask @ zone.js:402
runTask @ zone.js:159
invokeTask @ zone.js:483
D @ zone.js:1138
X @ zone.js:1181
ee @ zone.js:1202

onEnterEditMode | @ | DashboardScene.tsx:249
onApplyChange | @ | InspectJsonTab.tsx:123
```

**Environment**:
- SM plugin version: 1.15.2

**Comment by peterschretlen** [comment]
I think this has to do with this panel being a relatively complex scenes component that includes 2 datasources, many transformations, and a custom transform operator

https://github.com/grafana/synthetic-monitoring-app/blob/main/src/scenes/Summary/summaryTable.ts

**Comment by peterschretlen** [comment]
As an interim solution, would it make sense to remove/disable the Copy JSON button on the panel?

**Comment by peterschretlen** [comment]
Next action:
- disable the Copy JSON until we can fix this
- investigate what is involved to fix this properly, and open a followup issue.

**Comment by w1kman** [comment]
The problem has to do with Scenes.
Take ANY dashboard, go to ANY panel and open `Inspect > Panel JSON`, hit "Apply" (without modifying the JSON), see error.

**Workaround**
Enable "Edit mode" for the dashboard before trying to modify any panel JSON.

Awaiting response from Scenes folks.

**Comment by w1kman** [comment]
Blocked by https://github.com/grafana/grafana/issues/95482
